src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const SET_CATEGORY = 'SET_CATEGORY';
+
+const initialState = [];
+
+export const setCategory = (payload) => ({
+  type: SET_CATEGORY,
```

Comment:
- [ ] I would recommend you ensure the value of your action type returns follows the recommended ducks pattern as I have demonstrated above.

- [ ] **[OPTIONAL]** I would recommend you change this action type and creator to something more descriptive like `CHECK_STATUS` and `checkStatus`, since technically speaking, the action only checks for the categories message, not set it.